Traceback (most recent call last):
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/mshira/Dropbox/2024数理工学実験２/2024Shiraishi/.venv/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# タプルの要素を削除する
tuple4 = ("東京", "新宿", "渋谷","池袋","上野", "品川", "新橋", "お茶の水", "四谷", "中野", "吉祥寺")
del tuple4[7]
tuple4
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[9], line 3[0m
[1;32m      1[0m [38;5;66;03m# タプルの要素を削除する[39;00m
[1;32m      2[0m tuple4 [38;5;241m=[39m ([38;5;124m"[39m[38;5;124m東京[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m新宿[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m渋谷[39m[38;5;124m"[39m,[38;5;124m"[39m[38;5;124m池袋[39m[38;5;124m"[39m,[38;5;124m"[39m[38;5;124m上野[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m品川[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m新橋[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124mお茶の水[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m四谷[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m中野[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m吉祥寺[39m[38;5;124m"[39m)
[0;32m----> 3[0m [38;5;28;01mdel[39;00m [43mtuple4[49m[43m[[49m[38;5;241;43m7[39;49m[43m][49m
[1;32m      4[0m tuple4

[0;31mTypeError[0m: 'tuple' object doesn't support item deletion

